tauri-app/src-tauri/src/steam_manager.rs

```diff
@@ -31,7 +31,7 @@ impl SteamManager {
             // Steam on Flatpak
             home.join(".var/app/com.valvesoftware.Steam/.steam"),
```

Comment:
The above change breaks detection for Flatpak Steam.

As we are avoiding the symlink - you should update this to `.var/app/com.valvesoftware.Steam/.local/share/Steam` as well